### PR TITLE
chore: operations WorkStream FAQ additions

### PR DESCRIPTION
**Pull request by 0xApotheosis**
Re-opening https://github.com/shapeshift/docs/pull/47 from @Neverwas-dev so I can fix merge conflicts and other issues.

### Description

---

This is the initial push for the newly created FAQ section for the DAO repo and GitBook.

### Notice

---

- [x] Have you followed the guidelines in our Contributing guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

### Pull Request Type

---

 - [ ] 🐛 Bug fix (Non-breaking Change: Fixes an issue)
 - [x] 🛠️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ]  💅 New Feature (Breaking/Non-breaking Change)

### Notes

---

This was created in a few iterations on GitBook, I cleaned up most of the markdown trivial warnings.

### Risk

---

As of now just adding the structure and folders with the new files, not let linking from the root dir.

### CODE REVIEW

File: faq/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+description: About ShapeShift, the organization and it's current platform
```
Comment (gomesalexandre) [inline faq/README.md:2]:
```suggestion
description: About ShapeShift, the organization and its current platform
```

File: faq/about-shapeshift/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+description: Articles about our organization, platform, and operations
+cover: ../../.gitbook/assets/landingpage_hires.png
+coverY: 0
+---
+
+# 🦊 ShapeShift
+
+Thank you for being apart of ShapeShift
```
Comment (gomesalexandre) [inline faq/about-shapeshift/README.md:9]: 👀

File: faq/SUMMARY.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  * [🔄 Contribute](faq/about-shapeshift/contribute/README.md)
+    * [💰 Bounties](faq/about-shapeshift/contribute/bounties.md)
+    * [💻 Code Base](faq/about-shapeshift/contribute/codebase.md)
+  * [🔸 What Is A DEX?](faq/about-shapeshift/what-is-a-dex.md)
+  * [🎗 Important Links When Interacting with the ShapeShift DAO](faq/about-shapeshift/important-links-when-interacting-with-the-shapeshift-dao.md)
+  * [📡 FOX Current CEX/DEX Listings](faq/about-shapeshift/daos-current-cex-dex-listings.md)
+  * [🔤 What are Your Terms of Service?](faq/about-shapeshift/what-are-your-terms-of-service.md)
+  * [🗳 Governance](faq/about-shapeshift/governance.md)
+* [🌐 App.ShapeShift](faq/app.shapeshift/README.md)
+  * [🚀 What is app.ShapeShift.com?](faq/app.shapeshift/what-is-app.shapeshift.com.md)
```
Comment (gomesalexandre) [inline faq/SUMMARY.md:19]:
```suggestion
  * [🚀 What is app.shapeshift.com?](faq/app.shapeshift/what-is-app.shapeshift.com.md)
```

File: faq/SUMMARY.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* [🥳 FAQ's](README.md)
+
+## 📙 FAQ
+
+* [🦊 ShapeShift](faq/about-shapeshift/README.md)
+  * [🔮 The ShapeShift DAO](faq/about-shapeshift/the-shapeshift-dao.md)
+  * [🎨 ShapeShift Branding](faq/about-shapeshift/shapeshift-branding.md)
+  * [🔄 Contribute](faq/about-shapeshift/contribute/README.md)
+    * [💰 Bounties](faq/about-shapeshift/contribute/bounties.md)
+    * [💻 Code Base](faq/about-shapeshift/contribute/codebase.md)
```
Comment (gomesalexandre) [inline faq/SUMMARY.md:12]:
```suggestion
    * [💻 Codebase](faq/about-shapeshift/contribute/codebase.md)
```

File: faq/mobile.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# 📱 Mobile
+
+A new open-sourced Shapeshift Mobile App is here!.
```
Comment (gomesalexandre) [inline faq/mobile.md:3]:
```suggestion
A new open-sourced Shapeshift Mobile App is here!
```

**Review by gomesalexandre — APPROVED** [review]
Few typos spotted while reading through this, as comments for posteriority, since this is only the initial PR

**Comment by 0xApotheosis** [comment]
Thanks @gomesalexandre. I'll merge this as-is and then get a follow-up PR up with your suggestions 🐐

**Pull request merged**